gallery/src/components/NxDropdown/NxDropdownScrollingExample.tsx

```diff
@@ -9,50 +9,52 @@ import React from 'react';
 import { NxDropdown, useToggle } from '@sonatype/react-shared-components';
 
 function NxDropdownNavigationExample() {
-  const [isOpen, onToggleCollapse] = useToggle(false),
-      onClick = () => { alert('click'); };
+  const [isOpen, onToggleCollapse] = useToggle(false);
 
   return (
     <NxDropdown label="Scrolling - this label also triggers truncation"
                 isOpen={isOpen}
                 onToggleCollapse={onToggleCollapse}>
-      <a href="#" onClick={onClick} className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button">
         Text Link 1
       </a>
-      <a href="#" onClick={onClick} className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button">
         Text Link 2
       </a>
-      <a href="#" onClick={onClick} className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button">
         Text Link 3
       </a>
-      <a href="#" onClick={onClick} className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button">
         Text Link 4
       </a>
-      <a href="#" onClick={onClick} className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button">
         Text Link 5
       </a>
-      <a href="#" onClick={onClick} className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button">
         Text Link 6
       </a>
-      <a href="#" className="disabled nx-dropdown-button">
+      <a href="#/"
+         onClick={evt => { evt.preventDefault(); }}
+         className="disabled nx-dropdown-button"
+         aria-disabled="true">
```

Comment:
Aside from the href, the improvements on this element weren't actually things that came from the plugin; just my own tidying up.